### DISCUSSION TITLE
How to overwrite Pimcore Portal Search

### DISCUSSION THREAD

**Discussion by rushitkhunt1996** [pimcore/pimcore#15073] [Q&A / Support]
Hi All,

The Pimcore portal search is returns only published objects in the results, and we need an unpublished objects list also on search results.

Is there any way to do this?
![image](https://user-images.githubusercontent.com/60934941/235599995-4bae1af5-99f9-4195-9fb4-8b1b9cea1f7f.png)

**Comment by kingjia90** [comment]
Don't know but my 2 cents:

Wouldn't that cancel the purpose of the published/unpublished status?
Maybe you just need a custom field that mimic/represents a published status, i wouldn't change the behaviour of the "native" published/unpublished.

**Reply by rushitkhunt1996** [reply]
Yes, @kingjia90 Agreed with you, but the customer need publish/unpublish objects in the results so I thought let's do some customization if possible, otherwise I need to Conway them to use some flags with different solutions.
